src/main/java/org/opendevstack/provision/services/BitbucketAdapter.java

```diff
@@ -279,8 +289,10 @@ protected RepositoryData callCreateRepoApi(String projectKey, Repository repo)
                   + " - no response from endpoint, please check logs",
               repo.getName(), projectKey));
     }
-    setRepositoryPermissions(data, projectKey, ID_GROUPS, repo.getUserGroup());
-    setRepositoryPermissions(data, projectKey, ID_USERS, technicalUser);
+    setRepositoryAdminPermissions(data, projectKey, ID_GROUPS, repo.getUserGroup());
+    setRepositoryAdminPermissions(data, projectKey, ID_USERS, technicalUser);
+    setRepositoryPermissions(
```

Comment:
@stefanlack  - that one I am NOT ok with ... that means everyone has write access .. (and we removed that long time ago ..) why is this needed...?

Reply:
@clemensutschig because it is our philosophy for OpenDevStack@OPITZ and was requested by @rattermeyer .
But I can see your thoughts about that. We also had some internal discussions.

Suggested Solution: 
- Implementing an property `provision.scm.grantglobalwritetouser` (default value=false). 

By setting to true, the  line `setRepositoryPermissions(data, projectKey, ID_GROUPS, openDevStackUsersGroupName, REPOSITORY_PERMISSIONS.REPO_WRITE);` is executed, otherwise it is not executed


>

Reply:
Is this ok for you?

Reply:
@stefanlack .. there was a rule in the codebase.. if no special perm set.. then its ok... in case of special perm set.. no addtl write rules